specs/phase0/beacon-chain.md

```diff
@@ -1320,7 +1320,18 @@ def process_justification_and_finalization(state: BeaconState) -> None:
     # Skip FFG updates in the first two epochs to avoid corner cases that might result in modifying this stub.
     if get_current_epoch(state) <= GENESIS_EPOCH + 1:
         return
+    previous = get_matching_target_attestations(state, get_previous_epoch(state))
+    current = get_matching_target_attestations(state, get_current_epoch(state))
+    weigh_justification_and_finalization(
+        state, get_total_active_balance(state),
+        get_attesting_balance(state, previous),
+        get_attesting_balance(state, current))
+```
 
+```python
+def weigh_justification_and_finalization(state: BeaconState, total_active_balance: Gwei,
```

Comment:
I don't think spec ever does the two params on a multi-line function call like this. prefer to keep it standard so implementers get the visual queue

```suggestion
def weigh_justification_and_finalization(state: BeaconState,
                                         total_active_balance: Gwei,
```